### DISCUSSION TITLE
how do I download the app for my own usage?

### DISCUSSION THREAD

**Discussion by oferRounds** [insidegui/WWDC#716] [Q&A] [answered]
Couldn’t find it in the readme file

**Comment by insidegui** [comment, marked as answer]
Download the current release here: https://github.com/insidegui/WWDC/releases/download/7.5/WWDC_v7.5-1044.dmg

The most recent release can always be found here: https://github.com/insidegui/WWDC/releases/tag/7.5 (there is a link in the readme btw)

**Comment by oferRounds** [comment]
Thank you! I saw the link, but I couldn’t find any link to the dmg flile
